modules/animationqt/src/animationqtmodule.cpp

```diff
@@ -208,7 +208,12 @@ AnimationQtModule::AnimationQtModule(InviwoApplication* app)
                                      ControlSequenceEditor::classIdentifier());
 }
 
-AnimationQtModule::~AnimationQtModule() { unRegisterAll(); }
+AnimationQtModule::~AnimationQtModule() {
```

Comment:
Note this is only since this modules also owns the factory, other modules does not have to do this...

Reply:
adjusted comment in d1540e2